Shared/messages/MessageFeed.h

```diff
@@ -21,18 +21,32 @@
 #include <QObject>
 #include <QUrl>
 
+// MapsSDK headers
+#include "DynamicEntityDataSource.h"
+
+namespace Esri::ArcGISRuntime {
+
+class DynamicEntityDataSourceInfo;
+class DynamicEntity;
+
+}
+
 namespace Dsa {
 
+class Message;
 class MessagesOverlay;
 
-class MessageFeed : public QObject
+class MessageFeed : public Esri::ArcGISRuntime::DynamicEntityDataSource
```

Comment:
Is this class now exclusively for DynamicEntity logic? If it implements the `DynamicEntityDataSource` class, it shouldn't be responsible for anything else.